WhatsNewInCTSM5.3.md

```diff
@@ -1,53 +1,48 @@
-Purpose and description of changes since ctsm5.2.005
-----------------------------------------------------
+# What's new in CTSM 5.3 (tag `ctsm5.3.0`)
 
-Bring in updates needed for the CESM3.0 science capability/functionality "chill". Most importantly bringing
-in: CN Matrix to speed up spinup for the BGC model, updated surface datasets, updated Leung 2023 dust emissions,
-explicit Air Conditioning for the Urban model, updates to crop calendars. For clm6_0 physics these options are now
-default turned on in addition to Sturm snow, and excess ice.
+## Purpose and description of changes since CTSM 5.2 (tag `ctsm5.2.005`)
+- Adds CN Matrix method to speed up spinup for the BGC model.
+- Updates surface datasets.
+- Brings in new Leung 2023 dust emissions.
+- Adds explicit air conditioning for the urban model.
+- Updates crop calendars.
+- Updates fire model with various improvements, including default parameterization against climate reanalysis from CRU-JRA instead of GSWP.
+- FATES compsets can now be run with transient land use.
 
-Changes to CTSM Infrastructure:
-===============================
+These changes were needed ahead of the CESM3 capability/functionality "chill". For `clm6_0` physics these options are now turned on by default, in addition to Sturm snow and excess ice.
 
- - manage_externals removed and replaced by git-fleximod
- - Ability to handle CAM7 in LND_TUNING_MODE
+## Changes to CTSM infrastructure
+ - `manage_externals` removed and replaced by `git-fleximod`
+ - Ability to handle CAM7 in `LND_TUNING_MODE`
 
-Changes to CTSM Answers:
-========================
+## Changes to CTSM answers
```

Comment:
This section should mention new surface datasets and land-use timeseries files. Are those only used in `clm6_0` physics or all physics versions?

Reply:
All physics versions.